docs/userGuide/contentAuthoring.md

```diff
@@ -385,6 +385,10 @@ While authoring your website, you may want to have your own CSS or Javascript fi
 - Author your `<style>` elements for CSS files and `<link>` elements for Javascript files using HTML as shown below.
     - Ensure that your URLs start from the root directory, by using <code>{<span></span>{baseUrl}}/</code> when you are referencing your files.
 
+- Use `<head-top>` and `<head-bottom>` tags to control where your content will be placed.
+  - You must enclose your content in at least one tag for it to be placed in the `<head>`.
+  - If the other tag is not needed, you may omit it in your head file.

```

Comment:
Can we avoid this extra work to the author? i.e., normally added to the bottom but if these tags are present, they will be honored.

Reply:
Might get a bit confusing though if the user only uses `<head-bottom>` but not `<head-top>`.

```html
<script src="appearsToBeInsertedAtTop-butIsInsertedAtBottom.js" />

<head-bottom>
<script src="alwaysAtBottom.js" />
</head-bottom>
```

Reply:
May be we don't need the bottom tag at all?

Reply:
**Messy example**
```html
<script src="blahA"></script>
<!-- random comment -->
<script src="blahB"></script>
<head-top>
  <script src="kingOfScripts"></script>
</head-top>
<!-- random comment 2 -->
<script src="blahC"></script> 
```

Implementing just the `<head-top>` tag seems unbalanced, as the user can insert it anywhere they want :/

Reply:
**Common things should be _easy_ to do. Uncommon things should be _possible_ to do.** The common thing is adding to the bottom, which should be possible with the least effort.

Our instructions can be _Normally added to the bottom. If you want to add to the top, enclose them in a `<top>` tag_.

In the example we give, we can show the neat way to do it so that users do the same. But it's ok even if the top tag is in the middle. We just pick it up from the middle and add the rest to the bottom.